doc/manual/manual.tex

```diff
@@ -7934,9 +7918,31 @@ \subsubsection{Inner core convection}
 Figure~\ref{fig:inner-core-regimes} shows examples of the three regimes with $Ra=3000, \mathcal{P}=1000$ (plume convection), 
 $Ra=10^5, \mathcal{P}=0.01$ (translation), $Ra=10, \mathcal{P}=30$ (no convection).
 
+\begin{figure}[h]
+    \begin{center}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e0P0modif.png}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e2P-1modif.png}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e5P4modif.png}
+   \vspace{1cm}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e0P0modif.png}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e2P-1rescalemodif.png}
+    \includegraphics[width=0.25\linewidth]{cookbooks/inner_core_convection/Ra1e5P4rescalemodif.png}
+    \caption{Convection regimes in the inner core for different values of $Ra$ and $\mathcal{P}$. From left to right: no convection, translation, plume convection; the 2D slices of the top are with the default temperature scale while at the bottom an adaptative scale is used.}
+    \label{fig:inner-core-regimes}
```

Comment:
Nice! Would it be too much effort to also add velocity vectors somehow? I just think that would make a lot of sense, because then one would really see that one model is convecting, one translating, and one basically does not move at all.

Reply:
I guess we would need to go back to your initial figure for that... Or I can try to find the run somewhere on the harddrive. Let's see that Tuesday?